src/content/nft/index.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Non-fungible tokens (NFT)
+description: An overview of NFTs on Ethereum
+lang: en
+template: use-cases
+sidebar: true
+sidebarDepth: 2
+image: ../../assets/eth2/core.png
+summaryPoints:
+  [
+    "A way to turn anything unique into an Ethereum-based asset.",
```

Comment:
Maybe "A way to represent anything unique as an Ethereum-based asset." would be more clear.